spread.yaml

```diff
@@ -61,7 +61,7 @@ backends:
                 kernel: GRUB 2
                 workers: 4
             - fedora-25-64:
-                workers: 1
+                workers: 2
```

Comment:
Please limit this to 1 again. @niemeyer asked me to use just a single machine for both Fedora and openSUSE.